libbeat/processors/add_locale/add_locale.go

```diff
@@ -1,29 +1,99 @@
 package actions
 
 import (
+	"fmt"
+	"strings"
 	"time"
 
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/processors"
 )
 
-type addLocale struct{}
+type addLocale struct {
+	TimezoneFormat TimezoneFormat
+}
+
+type TimezoneFormat int
+
+const (
+	Abbrevation TimezoneFormat = iota
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const Abbrevation should have comment (or a comment on this block) or be unexported